gitpod-io/website
# ✨ Add New Customer story to enterprise page

## Description



Add the New Customer Story of Quizlet to the Enterprise page also 😀

![image](https://user-images.githubusercontent.com/55068936/224313057-4a2ba2cd-cdf0-4a09-9bd0-96c567d50814.png)


## How to test



Open [`/for/enterprise`](https://deploy-preview-3509--gitpod-kumquat.netlify.app/for/enterprise) & see :eyes:

## Release Notes



```release-note
NONE
```
